tc/sgx/trusted_worker_manager/enclave/kme/work_order_kme.edl

```diff
@@ -29,7 +29,7 @@ enclave {
 
         // outSerializedResponse is a base64 encoding of a JSON object
         // encrypted with the AES session key
-        public tcf_err_t ecall_GetSerializedResponseKME(
+        public tcf_err_t ecall_GetSerializedResponse(
             [out, size = inSerializedResponseSize] uint8_t* outSerializedResponse,
             size_t inSerializedResponseSize
             );
```

Comment:
Above is common. So we don't need work_order_kme.edl

Reply:
Deleted work_order_kme.edl file.